packages/scandipwa/src/ui/Button/Button.container.tsx

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable @scandipwa/scandipwa-guidelines/jsx-no-props-destruction */
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/scandipwa
+ * @link https://github.com/scandipwa/scandipwa
+ */
+
+import { ButtonHTMLAttributes, PureComponent } from 'react';
+
+import { FieldReactEvents } from 'Component/Field/Field.type';
+import { ReactElement } from 'Type/Common.type';
+
+import { ButtonContainerProps, ButtonContainerPropsKey } from './Button.type';
+
+/** @namespace ui/Button/Container */
+export class ButtonContainer extends PureComponent<ButtonContainerProps> {
+    static defaultProps: Partial<ButtonContainerProps> = {
+        mix: {},
+    };
+
+    containerProps(): Pick<ButtonContainerProps, ButtonContainerPropsKey> {
+        const { mix } = this.props;
+
+        return { mix };
+    }
+
+    render(): ReactElement {
+        const { children, attr, events } = this.props;
+
+        return (
+            <button
```

Comment:
I do not want `button` element to be styled by default, let's make a class for it? This way it can be link or a button...

Reply:
We can also allow passing `as` prop, which could be `Link` then a button could be link too 🤔